charts/sn-platform/templates/broker/broker-cluster.yaml

```diff
@@ -349,9 +349,12 @@ spec:
       authorizationEnabled: "true"
       authorizationProvider: "org.apache.pulsar.broker.authorization.PulsarAuthorizationProvider"
       {{- end }}
-      {{- if and .Values.broker.kop.enabled .Values.kop.auth.enabled }}
+      {{- if .Values.broker.kop.enabled }}
+      PULSAR_PREFIX_kafkaTransactionCoordinatorEnabled: "true"
```

Comment:
do we want to make it a flag or alway enable?

Reply:
Let me check with Yunze about any impact if always enable this config.

Reply:
@MarvinCai Just checked with Yunze, it's ok to keep this enabled.